deblur/workflow.py

```diff
@@ -438,16 +485,22 @@ def launch_workflow(seqs_fp, working_dir, read_error, mean_error, error_dist,
         filepath(s) to SortMeRNA indexed database for artifact removal
     negate: boolean
         discard all sequences aligning to the ref_fp database
-    threads: integer, optional
-        number of threads to use for SortMeRNA
+    threads_per_sample: integer, optional
```

Comment:
since the parallelization will be using threads, i prefer to keep this name to prevent confusion between threads for parallel sample processing and threads per sample.